src/main/java/eu/dissco/core/translator/terms/specimen/Modified.java

```diff
@@ -2,23 +2,38 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import eu.dissco.core.translator.terms.Term;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
 import java.util.List;
 
 public class Modified extends Term {
 
   public static final String TERM = "dcterms:modified";
+  private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern(
+      "yyyy-MM-dd'T'HH:mm:ss.SSSXXX").withZone(
```

Comment:
this date string is already in application configurations

Reply:
Yes, fixed